config/default/field.field.node.government_gazette.field_file.yml

```diff
@@ -0,0 +1,46 @@
+uuid: ff6ff698-cdbf-4238-bbc3-472cf6690edc
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_file
+    - node.type.government_gazette
+  module:
+    - file
+    - filefield_paths
+third_party_settings:
+  filefield_paths:
+    enabled: true
+    file_path:
+      value: ''
+      options:
+        slashes: false
+        pathauto: false
+        transliterate: false
+    redirect: false
+    retroactive_update: false
+    active_updating: false
+    file_name:
+      value: '[file:ffp-name-only-original].[file:ffp-extension-original]'
+      options:
+        slashes: false
+        pathauto: false
+        transliterate: false
+id: node.government_gazette.field_file
+field_name: field_file
+entity_type: node
+bundle: government_gazette
+label: 'Gazette download'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  file_directory: '[date:custom:Y]-[date:custom:m]'
+  file_extensions: txt
```

Comment:
I think this will need to be `pdf`. I'll update it if so.

Reply:
Ya, sorry i thought i changed it to txt doc docx xls pdf ppt pps odt ods odp rtf, but clearly didn't.